docker/single/Dockerfile.arm64

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:22.04
```

Comment:
Is it necessary to duplicate all of this, or would it be possible to use a build arg to combine both into a single Dockerfile?

Reply:
There are two packages we download that are architecture specific. I think there might be a way to do it with build args.

Reply:
Fixed this.